### DISCUSSION TITLE
为何又新建不了文章了

### DISCUSSION THREAD

**Discussion by abeginner-h** [Qexo/Qexo#232] [General]
[![xNSqGF.jpg](https://s1.ax1x.com/2022/10/11/xNSqGF.jpg)](https://imgse.com/i/xNSqGF)

**Comment by am-abudu** [comment]
请检查是不是没有模板文件scaffolds/post.md

我将在以后的版本去除这个问题

**Reply by am-abudu** [reply]
2bd674916f89f15fc5077ea290591cb365179ee9
